easybuild/easyconfigs/f/flex/flex-2.6.4-GCCcore-7.3.0.eb

```diff
@@ -14,11 +14,16 @@ toolchainopts = {'pic': True}
 
 source_urls = ['https://github.com/westes/flex/releases/download/v%(version)s/']
 sources = [SOURCELOWER_TAR_GZ]
-checksums = ['e87aae032bf07c26f85ac0ed3250998c37621d95f8bd748b31f15b33c45ee995']
+patches = ['flex-2.6.4_system_extension.patch']
+checksums = [
+    'e87aae032bf07c26f85ac0ed3250998c37621d95f8bd748b31f15b33c45ee995',  # flex-2.6.4.tar.gz
+    '2975201c6ec3687219cb1b7f85f04aee227ea67f0491ba16d50f0384a8e02630',  # flex-2.6.4_system_extension.patch
+]
 
 builddependencies = [
-    ('Bison', '3.0.4'),
-    ('help2man', '1.47.4'),
+    ('Bison', '3.0.5'),
+    ('help2man', '1.47.6'),
```

Comment:
@crubb bumping versions in existing easyconfigs is not going to work out, especially for low-level stuff like `flex`...